src/test/java/edu/oakland/test/display01/DisplayGpsInterfaceImplementationTests.java

```diff
@@ -0,0 +1,57 @@
+package edu.oakland.test.display01;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import edu.oakland.helper.admin.Satellite;
+import edu.oakland.helper.display01.SatelliteSignalCheckRequest;
+import edu.oakland.production.display01.DisplayGpsInterfaceImplementation;
+import edu.oakland.production.display01.DisplayGpsReceiver;
+import edu.oakland.test.display01.DisplayGpsReceiverStub;
+import java.util.Random;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+
+@DisplayName("Display GPS Interface Unit Tests")
```

Comment:
For standard log names, can we make this:
```suggestion
@DisplayName("DisplayGpsInterfaceImplementation Unit Tests")
```